### ISSUE TITLE
New Campaign design page 

### ISSUE THREAD

**Issue by swolf86** [podkrepi-bg/frontend#1072]
As a donor user I want to see less text as I don't usually read it, with the possibility to expand and read everything. Please bear in mind payment card on right to be changed ( submitted on separate issue ) 
Gallery will open once donor user clicks on the photos 
Like and share button under news section 

![Group 1335](https://user-images.githubusercontent.com/75036745/197404369-eb279529-e1a0-4b8c-9e1d-bddcdfb89b2e.png)
![Кампания скрит текст 2](https://user-images.githubusercontent.com/75036745/197404339-40668199-6910-4167-a82e-f13af6f1041d.jpg)
![Кампания скрит текст](https://user-images.githubusercontent.com/75036745/197404341-c9a9271f-cd08-4847-bfbb-1149b8fd7428.jpg)
![Group 1334](https://user-images.githubusercontent.com/75036745/197404366-ca491d85-c405-4282-8382-84241fad2b59.png)

**Comment by ani-kalpachka** [comment]
Some updates are added here: https://github.com/podkrepi-bg/frontend/pull/1326

What is left to be done:
- New gallery representation
- Read more / Read less button
- Improvements on the Messages
- Add Reports section
- Add News section

**Comment by swolf86** [comment]
New issue link to the this epic 
https://github.com/podkrepi-bg/frontend/issues/1399

**Comment by swolf86** [comment]
@igoychev  can 1072 be turned into EPIC 
Adding a few more issues 
wishes and donations 
https://github.com/podkrepi-bg/frontend/issues/1429
Tags 
https://github.com/podkrepi-bg/frontend/issues/1430
guarantor 1,2,4 
https://github.com/podkrepi-bg/frontend/issues/1431

**Comment by igoychev** [comment]
> @igoychev can 1072 be turned into EPIC

Yes, I have updated the title and main description, now containing the subtasks.

**Comment by ani-kalpachka** [comment]
For 'Add Organizer's profile / website' subtask I suggest we save the email instead of adding a website because if the organizer is a person he/she may not have a personal website.
Another option is to save the email for contact and add a website for the organizers that have one.
